src/App.css

```diff
@@ -1,22 +0,0 @@
-.App {
```

Comment:
Do we want to keep App.css in the src file, as we are keeping App.js there?

Reply:
I don't mind where it is located. Perhaps @Amaka202  and @saliainred  can weigh in on this too.